static/js/components/search/ModifiedSelectedFilter.js

```diff
@@ -13,6 +15,12 @@ export default class ModifiedSelectedFilter extends React.Component {
   }
 
   countryNameTranslations: Object = makeCountryNameTranslations()
+  countryNameList: string = _.values(this.countryNameTranslations)
+
+  isResidence = (labelKey: string) =>
+    R.or(_.includes(labelKey, "Country"), _.includes(labelKey, "Residence"))
+
+  isState = (labelKey: string) => _.indexOf(this.countryNameList, labelKey) > -1
```

Comment:
Why do we need this function? It seems like we just want to check the key, not the value

Reply:
label key for a state is a a country name, this Pull request fixes an issue which appears when user filters a company with name `ME`,  because `ME` is also state abbr. thats why `ModifiedSelectedFilter.js` was picking state full name instead of me

Thats why i am checking if country name exits in tranlations